docs/changelog.rst

```diff
@@ -16,7 +16,9 @@ The third digit is only for regressions.
 Changes:
 ^^^^^^^^
 
-- none yet
+- Add optional_ validator.
```

Comment:
I’m sorry I have to pick on this again but the changeling is where people learn about new things in a version so it should be more inviting.

Suggested alternative:

```
Add :func:`attr.validators.optional` that wraps other validators allowing attributes to be ``None``.
```